build.gradle

```diff
@@ -178,6 +178,17 @@ subprojects {
         }
     }
 
+    task copyMavenMetadataForDevelopment(type: Copy) {
+        from('build/tmp/publishMavenJavaPublicationToMavenLocal') {
+            rename 'module-maven-metadata.xml', 'maven-metadata.xml'
```

Comment:
is this actually renaming the file? Is that necessary if we just want to copy it?

Reply:
No, it isn't renaming the file in the source directory, but renames only during the copy. This is necessary, otherwise the copy task would write `module-maven-metadata.xml` to the target destination.

After this task runs, the original file in build/tmp/publishMavenJavaPublicationToMavenLocal still exists, e.g.`smithy-model/build/tmp/publishMavenJavaPublicationToMavenLocal/module-maven-metadata.xml`.